install-steps.md

```diff
@@ -1,193 +1,821 @@
+the OCP cluster on the 3 master and 2 worker nodes. For the purposes of this
+document, installing RHEL on the provision node is out of scope. However, options
+include, but not limited to, using a RHEL Satellite server, PXE, or installation
+media. 
+
+# Preparing the Provision node for OpenShift Install
+
+The following steps need to be performed in order to prepare the environment. 
+
+1. Login into the provision node via `ssh`
+2. Start `firewalld`, enable the `http` service, enable port 5000.
+   ~~~sh
+   systemctl start firewalld
+   firewall-cmd --zone=public --add-service=http --permanent
+   firewall-cmd --zone=public --permanent --add-service=http --permanent
+   firewall-cmd --add-port=5000/tcp --zone=libvirt  --permanent
+   firewall-cmd --add-port=5000/tcp --zone=public   --permanent
+   firewall-cmd --reload
+   ~~~
+3. Disable SELinux
+    ~~~sh
+    sed -i "s/=enforcing/=permissive/g" /etc/selinux/config
+    setenforce 0
+    ~~~
+4. Register your environment using `subscription-manager`
+   ~~~sh
+   subscription-manager register --username=<user> --password=<pass>
+   subscription-manager repos --enable=rhel-8-for-x86_64-appstream-rpms --enable=rhel-8-for-x86_64-baseos-rpms
+   ~~~
+4. Install the following packages 
+   ~~~sh
+   dnf install -y libvirt qemu-kvm mkisofs python3-devel python3-pip gcc ipmitool
+   ~~~
+5. Start and enable the `libvirtd` service
+   ~~~sh
+   systemctl start libvirtd
+   systemctl enable libvirtd --now
+   ~~~
+6. Create the `default` storage pool and start it.
+   ~~~sh
+   virsh pool-define-as --name default --type dir --target /var/lib/libvirt/images
+   virsh pool-start default
+   virsh pool-autostart default
+   ~~~
+7. Configure networking (this step can also be run from the console)
+    ~~~sh
+    # You will be disconnected, but reconnect via your ssh session after running
+    export PUB_CONN=<baremetal_nic_name>
+    export PROV_CONN=<prov_nic_name>
+    bash -x <<EOS
+    nmcli con delete "$PROV_CONN"
+    nmcli con add type bridge ifname provisioning autoconnect yes con-name provisioning stp off
+    nmcli con modify provisioning ipv4.addresses 172.22.0.1/24 ipv4.method manual
+    nmcli con add type bridge-slave autoconnect yes con-name "$PROV_CONN" ifname "$PROV_CONN" master provisioning
+    nmcli con delete "$PUB_CONN"
+    nmcli con add type bridge ifname baremetal autoconnect yes con-name baremetal stp off
+    nmcli con add type bridge-slave autoconnect yes con-name "$PUB_CONN" ifname "$PUB_CONN" master baremetal
+    systemctl restart NetworkManager
+    systemctl restart libvirtd
+    EOS
+    ~~~
+8. `ssh` back into your terminal session
+9. Verify the connection bridges have been properly created
+    ~~~sh
+    nmcli con show
+    ~~~
+10. Create a user (i.e `kni`) to deploy as non-root
+    ~~~sh
+    echo "kni ALL=(root) NOPASSWD:ALL" | tee -a /etc/sudoers.d/kni
+    chmod 0440 /etc/sudoers.d/kni
```

Comment:
This needs to be one of the first steps done and then make all the other commands use `sudo`